src/Motion.js

```diff
@@ -110,7 +117,7 @@ const Motion = createReactClass({
         return;
       }
 
-      this.wasAnimating = true;
+      this.wasAnimating = false;
```

Comment:
I changed this to avoid flow check error(current error message on travis), but I guess it causes karma error... I'm not farmiliar with Flow and internal logic of this file, Could you give me an idea how to fix this flow error?